pkg/scripts/install.sh

```diff
@@ -120,13 +118,13 @@ create_tfstate_resources() {
   echo "Checking the terraform state."
   cd "${SCRIPT_PATH}/../tfstate"
   set +e
-  aws s3api head-bucket --bucket "${S3_BUCKET}"
+  aws s3api head-bucket --bucket "${S3_BUCKET}" 2>/dev/null
   S3_BUCKET_EXISTS=$?
   set -e
   if [ ${S3_BUCKET_EXISTS} -eq 0 ]
   then
     echo "S3 bucket '${S3_BUCKET}' already exists."
-    cd "${CURRENT_PATH}"
+    cd "${ROOT_PATH}"
```

Comment:
again this is not needed. The intention is user can run the script from any folder, the scripts will run and after run returns to the same folder. I mean we should not change the current folder as result of running the script.

Reply:
The `cd` was there, I just did the rename from `CURRENT_PATH` which is not self-explanatory to `ROOT_PATH` which refers to the root folder.

Reply:
New version of the install script has no `cd` command at all. I removed all.